python/nav/portadmin/napalm/juniper.py

```diff
@@ -27,6 +27,7 @@
 from operator import attrgetter
 from typing import List, Any, Dict, Tuple, Sequence
 
+from django.template.loader import get_template
```

Comment:
With templates becoming this important we should have a better way of testing them. For finding which lines of a template is tested there's https://pypi.org/project/django-coverage-plugin/.

The minimum should be to check all templates for syntax errors, for instance with `TemplateSyntaxError`, but I don't know how much you'd catch without sending in any context. More research needed.

Reply:
We already syntax test all templates, at least.

However, I'm not 100% sure template files in app directories are checked...

Reply:
Well, indeed they are :)
https://ci.nav.uninett.no/job/nav-pipeline/job/PR-2204/1/testReport/tests.integration.web/templatesyntax_test/test_template_syntax_portadmin_junos_disable_interface_djt_/